rero/rero-ils
# User interface: menu list

* BETTER: select menu list in editor
* closes #70
* closes #126
* closes #45

to test:
`./scripts/bootstrap`
`./scripts/setup`

verify creation and edit of:
* item
* location
* patron
